nextflow.config

```diff
@@ -32,4 +32,16 @@ profiles {
     singularity.enabled = true
     singularity.autoMounts = true
   }
-}
\ No newline at end of file
+  
+  k8s {
```

Comment:
Hi Vinay, thank you for this.

Has this been tested in an EBI Kubernetes already?
Do we need to provide any additional Kubernetes cluster configuration to make the pipeline aware, unlike lsf or slurm which are available to the cluster already?

Reply:
Hi @vinay-ebi , this appears to be specific configuration and would it probably better in the project/team specific configuration unless we have a generic EBI k8s cluster